src/services/sessions.js

```diff
@@ -234,7 +249,13 @@ module.exports = class SessionsHelper {
 					responseCode: 'CLIENT_ERROR',
 				})
 			}
-
+			if (!sessionDetail.created_by !== userId) {
+				return common.failureResponse({
+					message: 'CANNOT_EDIT_DELETE_SESSION_CREATED_BY_MANAGER',
```

Comment:
@nevil-mathew Let this be a generic message, cannot edit the session, manager or not we need not tell